components/brave_wallet_ui/panel/container.tsx

```diff
@@ -381,6 +383,18 @@ function Container (props: Props) {
     // Logic here to cancel connecting your hardware wallet
   }
 
+  const removeSitePermission = (origin: string, address: string) => {
+    props.walletActions.removeSitePermission({ origin: origin, account: address })
+  }
+
+  const isConnectedToSite = React.useMemo((): boolean => {
+    if (activeOrigin === 'chrome://wallet/') {
```

Comment:
`'chrome://wallet/'` was used somewhere else too, so can we move this to a common place and give it a meaningful name?